omnigibson/transition_rules.py

```diff
@@ -722,6 +726,10 @@ def transition(cls, object_candidates):
             # Object parts offset annotation are w.r.t the base link of the whole object.
             pos, orn = sliceable_obj.get_position_orientation()
 
+            # If it has no parts, silently fail
```

Comment:
in general, should this edge case occur? i presume it happens now because we don't have all of our assets ready, but once we do, it should never happen, right? should we print out a warning so it's explicit when this should occur?

Reply:
Unfortunately this will keep happening - a lot of sliceable-annotated objects in the dataset will never become sliceable (we just don't have the bandwidth). All we are guaranteeing is that at least one object of the category will be sliceable.